Tests/Editor/CourseWindow/BaseCourseWindowTest.cs

```diff
@@ -70,11 +64,13 @@ protected override CourseWindow Given()
 
             GlobalEditorHandler.SetStrategy(new EmptyTestStrategy());
 
+            GUIUtility.keyboardControl = 0;
```

Comment:
Doesn't this happen automatically before?

Reply:
No, I don't think so.